### PR TITLE
[ACM-83] add new 2021 call for papers content

### PR DESCRIPTION
**Pull request by getsarker**
Update call for papers page with current 2021 information as provided in the link below:

https://docs.google.com/document/d/1lpt0J268YKNHdscFra6qJm6hwZZqFYxKa-HcZHV1wY0/edit#heading=h.9goj9zkzjmzk

### CODE REVIEW

File: templates/content/call-for-papers-track-2.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+Computer software submissions should describe the intended use for the software, justify the need for the software, and provide executable examples for other researchers. Software submissions should directly support a healthcare application. Examples include code for summarizing the demographics of a study cohort [4], deriving meaningful clinical concepts from electronic health records [5], and natural language processing tools specifically designed for clinical text [6, 7]. All computer software submissions must be open source and released under a suitable open source license. Computer software should adhere to best practices in software development where possible, including the use of unit tests, continuous integration, and diligent documentation of component design and purpose [8].
+
+4 Pollard TJ, Johnson AE, Raffa JD, Mark RG. tableone: An open source Python package for producing summary statistics for research papers. JAMIA Open. 2018 May 23;1(1):26-31.
```
Comment (corcra) [inline templates/content/call-for-papers-track-2.md:16]: The formatting around these references is off, should be `[4] Pollard...` I think? And I guess it should start at 1.

File: templates/content/call-for-papers-track-2.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+- New tools or comprehensive benchmarks for machine learning for healthcare.
+    * Wang, Shirly, et al. "MIMIC-Extract: A Data Extraction, Preprocessing, and Representation Pipeline for MIMIC-III." Machine Learning for Healthcare, 2019.
+- Development of Scalable Systems for Processing Data in Practice (demonstrating, e.g., concern for multi-modality, runtime, robustness, etc., as guided by a clinical use case):
+    * Xu, Yanbo, et al. "Raim: Recurrent attentive and intensive model of multimodal patient monitoring data." Proceedings of the 24th ACM SIGKDD International Conference on Knowledge Discovery & Data Mining. ACM, 2018.
+- Bridging the deployment gap
+    * Tonekaboni, Sana, et al. "What Clinicians Want: Contextualizing Explainable Machine Learning for Clinical End Use." Machine Learning for Healthcare (2019)
+- Remote, Wearable, and Telehealth [explicit callout for public health?]
+    * [https://ieeexplore.ieee.org/document/9022981](https://ieeexplore.ieee.org/document/9022981) [perhaps?]
```
Comment (corcra) [inline templates/content/call-for-papers-track-2.md:46]: This line and the previous one look unfinished?

File: templates/call-for-papers.html [hunk 3]
```diff
@@ -6,13 +6,65 @@
...
-
-{% block content %}
 <div class="page-header text-center">
   <h1>Call for Papers</h1>
 </div>
+{{ components.tabs([("author-info", "Author Information", "active"),
+                    ("track-1", "Track 1: Machine Learning", ""),
+                    ("track-2", "Track 2: Applications and Practice", ""),
+                    ("track-3", "Track 3: Impact and Society", ""),
+                    ("chil-2020-summary", "CHIL 2020 Summary", ""),
```
Comment (corcra) [inline templates/call-for-papers.html:16]: "CFP 2020" might be better (although I realise this is temporary)

**Review by corcra — CHANGES_REQUESTED** [review]
Looks fine, some of the content/formatting needs fixing/checking though.

**Comment by corcra** [comment]
Just adding a comment here - the content of the CFPs aren't actually finalised yet, so we need to wait for confirmation from the track chairs/Marzyeh that these are ready to go live.

File: templates/content/call-for-papers-track-2.md (hunk 1 above)
Comment (getsarker) [reply templates/content/call-for-papers-track-2.md]: Content updated.

File: templates/call-for-papers.html (hunk 3 above)
Comment (getsarker) [reply templates/call-for-papers.html]: Removed in favour of highlights page.

**Pull request merged**